google/cloud/bigtable/backup.py

```diff
@@ -0,0 +1,332 @@
+        ).match(backup_pb.name)
+        if match is None:
+            raise ValueError(
+                "Backup protobuf name was not in the expected format.", backup_pb.name
+            )
+        if match.group("project") != instance._client.project:
+            raise ValueError(
+                "Project ID of the Backup does not match the Project ID "
+                "of the instance's client"
+            )
+
+        instance_id = match.group("instance_id")
+        if instance_id != instance.instance_id:
+            raise ValueError(
+                "Instance ID of the Backup does not match the Instance ID "
+                "of the instance"
+            )
+        backup_id = match.group("backup_id")
+        return cls(backup_id, instance)
+
+    def create(self, cluster_id=None):
+        """Creates this backup within its instance.
+
+        :type cluster_id: str
+        :param cluster_id: (Optional) The ID of the Cluster for the newly
+                           created Backup.
+
+        :rtype: :class:`~google.api_core.operation.Operation`
+        :returns: :class:`~google.cloud.bigtable_admin_v2.types._OperationFuture`
+                  instance, to be used to poll the status of the 'create' request
+        :raises Conflict: if the Backup already exists
+        :raises NotFound: if the Instance owning the Backup does not exist
+        :raises BadRequest: if the `table` or `expire_time` values are invalid,
+                            or `expire_time` is not set
+        """
+        if not self._expire_time:
+            raise ValueError('"expire_time" parameter must be set')
+            # TODO: Consider implementing a method that sets a default value of
+            #  `expire_time`, e.g. 1 week from the creation of the Backup.
+        if not self.table_id:
+            raise ValueError('"table" parameter must be set')
+
+        if cluster_id:
+            self._cluster = cluster_id
+
+        if not self._cluster:
+            raise ValueError('"cluster" parameter must be set')
+
+        backup = {
+            "source_table": self.source_table,
+            "expire_time": _datetime_to_pb_timestamp(self.expire_time),
+        }
+
+        api = self._instance._client.table_admin_client
+        return api.create_backup(self.parent, self.backup_id, backup)
+
+    def get(self):
+        """Retrieves metadata of a pending or completed Backup.
+
+        :returns: An instance of
+                 :class:`~google.cloud.bigtable_admin_v2.types.Backup`
+
+        :raises google.api_core.exceptions.GoogleAPICallError: If the request
+                failed for any reason.
+        :raises google.api_core.exceptions.RetryError: If the request failed
+                due to a retryable error and retry attempts failed.
+        :raises ValueError: If the parameters are invalid.
+        """
+        api = self._instance._client.table_admin_client
+        try:
+            return api.get_backup(self.name)
```

Comment:
if not updating `self` (refer to comment above), should return `Backup` instance, not protobuf message `(self.from_pb(api.get_backup(self.name), self._instance))`